consul/plan.sh

```diff
@@ -0,0 +1,39 @@
+pkg_origin=core
+pkg_name=consul
+pkg_version=0.7.0
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=("Apache 2")
+pkg_upstream=https://www.consul.io/
+pkg_source=https://releases.hashicorp.com/${pkg_name}/${pkg_version}/${pkg_name}_${pkg_version}_linux_amd64.zip
+pkg_shasum=b350591af10d7d23514ebaa0565638539900cdb3aaa048f077217c4c46653dd8
+pkg_filename=${pkg_name}-${pkg_version}.zip
```

Comment:
`pkg_filename=${pkg_name}_${pkg_version}_linux_amd64.zip` will obviate the need for an overridden `do_download()` and the build dependency on wget.